semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/operatorextension/package.scala

```diff
@@ -4,76 +4,82 @@ import gremlin.scala.GremlinScala
 import io.shiftleft.codepropertygraph.Cpg
 import io.shiftleft.codepropertygraph.generated.Operators
 import io.shiftleft.codepropertygraph.generated.nodes
+import io.shiftleft.codepropertygraph.generated.nodes.Call
+import io.shiftleft.overflowdb.OdbGraph
 
 package object operatorextension {
+  class Assignment(call: Call) extends Call(call.graph.asInstanceOf[OdbGraph], call.id)
+  class Arithmetic(call: Call) extends nodes.Call(call.graph.asInstanceOf[OdbGraph], call.id)
+  class ArrayAccess(call: Call) extends nodes.Call(call.graph.asInstanceOf[OdbGraph], call.id)
```

Comment:
As long as the Traversal classes do not have names indicating them being traversals, I suggest we but these node types into a `nodes` namespace like the generated nodes and only use them via `nodes.XXX`. That way it is directly visible whether we talk about nodes or traversals.

Reply:
Good idea, will do